static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -0,0 +1,21 @@
+# How to update your pipeline by editing dvc files
+
+> 1)using `dvc move` to edit file paths of outputs or dependencies.
+> 2)editting by hand the outputs paths.
+> 3)`dvc run` to make edits in the dependency files and `dvc commit` to save
+> changes.
+
+If your source file is a data file, then use dvc move:
+
+`dvc move` is used if a file or a directory has been added to DVC with `dvc 
+add`. The source file or directory is moved to its destination path, renamed 
+and corresponding DVC-file is updated.
+
+If your source file isn't data file then use `dvc run`. Then add the new file
```

Comment:
I'm not sure I understand this ... why `dvc add` after `dvc run`?

Reply:
My bad! I have changed it.